m02_setup/02_git.rst

```diff
@@ -83,13 +84,12 @@ computer and the Aalto Gitlab computer safely. It is required for making
 the commands ``git pull`` and ``git push`` work.
 
 Click on your profile picture circle on the top-right corner of the
-page and choose "settings".
+page and choose "settings". Next, on the left sidebar, click :guilabel:`&SSH`  "SSH Keys" on the sidebar on the left.
```

Comment:
What should the `guilabel` (SSH) look like? In A+, it is not rendered well. Should it be a graphical icon or something?